### PR TITLE
GCP IAM Data Connector

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes

  - add GCP IAM Data Connector

### CODE REVIEW

**Review by v-rucdu — DISMISSED** [review]
Looks good

File: Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
+{
+    "id": "GCPIAMDataConnector",
+    "title": "Google Cloud Platform IAM (Preview)",
```
Comment (v-jayakal) [inline Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json:3]: Can you please remove Preview from title?

File: Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    "publisher": "Google Cloud Platform",
```
Comment (v-jayakal) [inline Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json:4]: Can you tell us whether the Publisher is "Google Cloud Platform" or "Google"? For 'GoogleWorkspaceReportsAPI', publisher was 'Google', hence asking.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json (hunk 1 above)
Comment (vu-socprime) [reply Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json]: updated

File: Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json (hunk 2 above)
Comment (vu-socprime) [reply Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json]: I think it is better to use "Google" here. File updated. Thanks

**Review by elforb — CHANGES_REQUESTED** [review]
Hi, one issue related to solutions.

The Cisco SEG update posted recently has an issue in the connector.

In particular, the sample query contained within the connector was not updated along with the  rest of the schema, and thus the sample query fails on a missing field, `SrcIpAddr`.
image

Could you look into this and correct the query?

**Comment by v-jayakal** [comment]
@vu-socprime , can you please help us in resolving the Cisco SEG sample query? Thanks.

**Comment by vu-socprime** [comment]
> @vu-socprime , can you please help us in resolving the Cisco SEG sample query? Thanks.

@elforb @v-jayakal Fixed in PR https://github.com/Azure/Azure-Sentinel/pull/2539

**Comment by v-rucdu** [comment]
@oshezaf : Could you please review the parser

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**